docs/user-guide/host-settings.md

```diff
@@ -1,13 +1,14 @@
 # Host Settings
-This page contains settings for Linux servers, it agnostic of the Linux
```

Comment:
Please have a meaningful commit description for this commit.  Maybe something like "minor edits, better formatting for console code sections"